app/views/hyrax/my/_facet_limit.html.erb

```diff
@@ -1,4 +1,4 @@
-  <ul class="dropdown-menu facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
+  <ul class="<%= 'dropdown-menu ' if request.format.html? %>facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
```

Comment:
We're in an HTML template.  So how would we get here if the request format wasn't HTML?